jax/_src/util.py

```diff
@@ -181,7 +181,7 @@ def curry(f):
   >>> curry(f)(2, 3, 4, 5)()
   26
   """
-  return partial(partial, f)
+  return wraps(f)(partial(partial, f))
```

Comment:
This was necessary because `transformation` and `transformation_with_aux` are curried, and otherwise have mal-formed docstrings.